.scripts/deploy-assistify-chat.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
*NO PR*

This file holds the instructions where and how to package which artifacts in order to get them running on our servers